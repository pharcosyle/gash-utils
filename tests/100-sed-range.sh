input='1 bar
2 bar
3 bar
4 bar
5 bar'

echo "$input" | \sed '2,4s/a/x/'
