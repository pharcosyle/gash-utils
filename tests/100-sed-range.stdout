1 bar
2 bxr
3 bxr
4 bxr
5 bar
